instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+const Comment = props => {
+  return (
+    <div className='comments'>
+      <p><span className='username'>{props.comment.username}</span> <span>{props.comment.text} </span></p>
+    </div>
+  )
+}
+
+Comment.propTypes = {
```

Comment:
If your proptypes here aren't `.isRequired` then you will need to specify defaultProps